modules/RTC/TraceablePeerConnection.js

```diff
@@ -1658,6 +1658,9 @@ TraceablePeerConnection.prototype.replaceTrack = function(oldTrack, newTrack) {
  * changed and the renegotiation is required or <tt>false</tt> otherwise.
  */
 TraceablePeerConnection.prototype.removeTrackMute = function(localTrack) {
+    if (browser.usesUnifiedPlan()) {
```

Comment:
why is this call moved above the _assertTrackBelongs? a comment would be helpful

Reply:
Please refer to the comment below